test/cypress/pages/obtener-respuestas/buscar.js

```diff
@@ -0,0 +1,28 @@
+export class ObtenerRespuestasBuscar {
+
+  open() {
+    cy.visit( '/es/obtener-respuestas/' );
+  }
+
+  enter( term ) {
+    cy.get( '#o-search-bar_query' ).type( term );
+  }
+
+  autocomplete() {
+    return cy.get( '.m-autocomplete_results' );
+  }
+
+  search() {
+    cy.get( 'form[action="/es/obtener-respuestas/buscar/"]' ).first().within( () => {
+      cy.get( '.a-btn' ).click();
+    } );
```

Comment:
One other simplification I forgot to suggest:

```suggestion
    cy.get( '.o-search-bar .a-btn' ).click();
```

Reply:
Good suggestions